src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 40 * days;
+
+  if (days >= 7) {
+    rentalCost -= 50;
+  } else if (days >= 3) {
+    rentalCost -= 20;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return rentalCost - 50;
  }
  
  if (days >= 3) {
    return rentalCost - 20;
  }
```
You can just immediately return the final price to avoid "else if" and make your code a bit cleaner